### ISSUE TITLE
Overhead Lines

### ISSUE THREAD

**Issue by NitroxydeX** [AlexIIL/TrainCraft-Ideas#5]
Would be nice if the Overhead Lines could be reimplemented.

Requirement: Mod that working with RF e.g. Buildcraft Energy

Overhead Line Turns would be needed for this.

**Comment by pietrodevo** [comment]
Yes, also EU would be nice, but here the choice is about long-term compat and most used energy system.
A very interesting example about how overhead line can be worked is in IHL tools & machines addon for industrialcraft, where there are "flexible cables" that works like the classic IC cables but are not blocks like them. Player can put some insulators (where actually cables are fixed) over the rails of the to let cables rotate with tracks

**Comment by AlexIIL** [comment]
Overhead cables (if they are implemented) will be powered by RF as that's what BuildCraft uses. (Just for context I am maintaining the 1.8.9 version of BuildCraft so compat with BC is more likely than any other mod).

**Comment by NitroxydeX** [comment]
I think Industrialcrafts EU-system isn't that good as RF. Look at Buildcraft it changed from MJ to RF and there couldn't be a better system than RF.

So @AlexIIL says it: If it get's implemented it's powered by RF. At least, there could be some Converter for it like Mekanism integrated it into their cables.
